xamarin/Xamarin.Forms
# [Android] SwipeView using RefreshView and Expander

### Description of Change ###

Force close the SwipeView tapping the content even using an Expander and launching the Expander Tapped event.

### Issues Resolved ### 

- fixes #11349 

### API Changes ###

 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

![fix11349](https://user-images.githubusercontent.com/6755973/91558149-13ec9900-e936-11ea-9c1a-687216f493ae.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 11349. Open and close the SwipeView, expand and collapse the Expander and try to refresh using the RefreshView. If everything is working as expected, the test has passed.

### PR Checklist ###
